app/views/speech/show.html.erb

```diff
@@ -1,37 +1,61 @@
+<% content_for :title do %>
+  <%= @content_item.title %> - GOV.UK
+<% end %>
 
 <% content_for :extra_headers do %>
-   <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: @content_item.to_h, schema: :article } %>
+   <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: @content_item.to_h, schema: :news_article } %>
+   <meta name="description" content="<%= strip_tags(@content_item.description) %>">
 <% end %>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <%= render 'govuk_publishing_components/components/title', {
-        title: content_item.title,
-        context: I18n.t("csv_preview.document_type.#{content_item.document_type}", count: 1),
-        context_locale: t_locale_fallback("csv_preview.document_type.#{content_item.document_type}", count: 1),
-        average_title_length: "long",
-      } %>
-  </div>
-  <%= render 'shared/translations' %>
-  <div class="govuk-grid-column-two-thirds">
-    <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
+<main role="main" id="content" class="speech" <%= lang_attribute(@content_item.locale) %>>
```

Comment:
The <main> tag has been globally included now (https://github.com/alphagov/frontend/pull/4414) so you don't need to include it here in the view.